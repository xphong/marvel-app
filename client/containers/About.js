import React from 'react';

const About = () => (
  <div className="fadeIn-page-animation">
    <h1>About</h1>
    <h4>Marvel Search App</h4>
    <ul>
      <li>Searches the Marvel database for the characters, power levels</li>
      <li>Built using <a href="https://github.com/xphong/simple-react-redux-starter" target="_blank" rel="noopener noreferrer">simple-react-redux-starter</a> and the <a href="http://developer.marvel.com/" target="_blank" rel="noopener noreferrer">Marvel API</a></li>
      <li><a href="https://github.com/xphong/marvel-react-app" target="_blank" rel="noopener noreferrer">Github Link</a></li>
      <li><a href="http://marvel.com" target="_blank" rel="noopener noreferrer">Data provided by Marvel. Â© {new Date().getFullYear()} Marvel</a></li>
    </ul>
  </div>
);

export default About;
