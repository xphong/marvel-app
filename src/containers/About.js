import React, { Component } from 'react';

export default class About extends Component {
  render() {
    return (
      <div className='fadeIn-page-animation'>
        <h1>About</h1>
        <h4>Marvel Search App</h4>
        <ul>
          <li>Searches the Marvel database for the characters, power levels</li>
          <li>Built using <a href='https://github.com/xphong/simple-react-redux-starter' target='_blank'>simple-react-redux-starter</a> and the <a href='http://developer.marvel.com/' target='_blank'>Marvel API</a></li>
          <li><a href='https://github.com/xphong/marvel-react-app' target='_blank'>Github Link</a></li>
          <li><a href='http://marvel.com' target='_blank'>Data provided by Marvel. Â© {new Date().getFullYear()} Marvel</a></li>
        </ul>
      </div>
    );
  }
}
