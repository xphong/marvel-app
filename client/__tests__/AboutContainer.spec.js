import React from 'react';
import ReactDOM from 'react-dom';

import About from '../containers/About';

describe('About Container', () => {
  it('should render About', () => {
    const wrapper = shallow(
      <About />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
