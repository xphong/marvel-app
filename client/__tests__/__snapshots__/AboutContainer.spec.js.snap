// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Container should render About 1`] = `
<div
  className="fadeIn-page-animation"
>
  <h1>
    About
  </h1>
  <h4>
    Marvel Search App
  </h4>
  <ul>
    <li>
      Searches the Marvel database for the characters, power levels
    </li>
    <li>
      Built using 
      <a
        href="https://github.com/xphong/simple-react-redux-starter"
        target="_blank"
      >
        simple-react-redux-starter
      </a>
       and the 
      <a
        href="http://developer.marvel.com/"
        target="_blank"
      >
        Marvel API
      </a>
    </li>
    <li>
      <a
        href="https://github.com/xphong/marvel-react-app"
        target="_blank"
      >
        Github Link
      </a>
    </li>
    <li>
      <a
        href="http://marvel.com"
        target="_blank"
      >
        Data provided by Marvel. Â© 
        2018
         Marvel
      </a>
    </li>
  </ul>
</div>
`;
